content/aips/STMATIC-MATICX-LDO-SD-EMISSION-ADMIN-UPDATE-AAVE-V3-POLYGON.md

```diff
@@ -0,0 +1,69 @@
+author: Llamax - Matthew Graham & defijesus
+discussion: https://governance.aave.com/t/arc-ldo-emission-admin-for-polygon-v3-liquidity-pool/10575
+https://governance.aave.com/t/arc-sd-emission-admin-for-polygon-v3-liquidity-pool/10658
+https://governance.aave.com/t/arc-stmatic-maticx-emission-admin-for-polygon-v3-liquidity-pool/10632
+shortDescription: Enable Lido Commununity, Stader Labs and Polygon Foundation to distribute rewards across the Polygon v3 Liquidity Pool
+created: 2022-12-05
+---
+
+# Summary
+
+Assign the Emission_Admin role for the LDO, stMATIC, MaticX and SD tokens on the Polygon v3 deployment.  
+
+# Abstract
+
+The following teams seek to distribute rewards across the Polygon v3 Liquidity Pool:
+
+* LDO by the Liquidity Mining Committee to support the adoption of stMATIC
+* SD by Stader Labs to support the adoption of MaticX
+* stMATIC & MaticX by Polygon Foundation to adoption of staked MATIC receipt tokens 
+
+In order to achieve this, the EmissionManager contract needs to assign an owner to the Emission_Admin permission allowing LDO, SD, stMATIC, and MaticX rewards to be distributed.
+
+# Motivation
+
+The motivation of the proposal is to enable rewards to be distributed on the Polygon v3 Liquidity Pool.
+
+The Lido Liquidity Mining Committee determines the emission rate and duration at which LDO rewards are offered. Standard practice is for the council to meet and determine the next 30 days of LDO rewards on a rolling basis. This means each month, the committee will determine if to extend the LDO rewards, or change the amount of LDO being offered across the Polygon v3 Liquidity Pool.
+
+The Stader Labs team will determine the emission rate and duration SD rewards are offered. This will likely be influenced by the Aave DAO’s ability to adjust the wMATIC and MaticX risk parameters that enable users to generate a return from leverage the MaticX / wMATIC loop.
+
+The Polygon Foundation team will determine the emission rate and duration at which stMATIC and MaticX rewards are offered. This will likely be influenced by Aave DAO’s ability to adjust the wMATIC, stMATIC and MaticX risk parameters that enable users to generate a return from leverage the stMATIC / wMATIC and MaticX/wMATIC loop.
+
+The Emission_Admin is not limited to a specified Reserve and is linked to the token being distributed across the Liquidity Pool. The EmissionManager determines the Emission_Admin. The Emission_Admin can distribute said token anywhere across the Aave v3 Polygon Liquidity Pool.
+
+# Specification
+
+This proposal assigns the Emission_Admin:
+
+LDO Emission Admin: `matic:0x87D93d9B2C672bf9c9642d853a8682546a5012B5`
+SD Emission Admin: `matic:0x51358004cFe135E64453d7F6a0dC433CAba09A2a`
+stMATIC & MaticX Emission Admin: `matic:0x0c54a0BCCF5079478a144dBae1AFcb4FEdf7b263`
+
+# Implementation
+
+setEmissionAdmin(address reward, address admin)
+
+LDO Address Reward: `matic:0xC3C7d422809852031b44ab29EEC9F1EfF2A58756`
```

Comment:
would move `Reward` to `Sepecification` section - doesn't really make sense to separate imo.
At least i don't understand what the difference is.